streaming_form_data/_parser.pyx

```diff
@@ -169,6 +174,9 @@ cdef class _Parser:
 
         self._leftover_buffer = None
 
+        self.strict = strict
+        self.unknown_part = ''
```

Comment:
```suggestion
        self.unexpected_part_name = ''
```

It's a nitpick but in the rest of the parser we're using `expected_parts` terminology, so just to make things consistent. :)